spark-cassandra-connector-embedded/src/main/scala/com/datastax/spark/connector/embedded/EmbeddedCassandra.scala

```diff
@@ -9,6 +9,7 @@ import java.net.InetAddress
 trait EmbeddedCassandra {
 
   def cassandraHost = EmbeddedCassandra.cassandraHost
+  def secondCassandraHost = SecondEmbeddedCassandra.cassandraHost
```

Comment:
Nothing is calling this in your changes. I would not put a call or notion of a second in the base.